modules/gdscript/gdscript_utility_callable.cpp

```diff
@@ -80,6 +80,22 @@ ObjectID GDScriptUtilityCallable::get_object() const {
 	return ObjectID();
 }
 
+int GDScriptUtilityCallable::get_argument_count(bool &r_is_valid) const {
+	switch (type) {
+		case TYPE_INVALID:
+			r_is_valid = false;
+			return 0;
+		case TYPE_GLOBAL:
+			r_is_valid = true;
+			return Variant::get_utility_function_argument_count(function_name);
+		case TYPE_GDSCRIPT:
+			r_is_valid = true;
+			return GDScriptUtilityFunctions::get_function_argument_count(function_name);
+		default:
+			ERR_FAIL_V_MSG(0, "Invalid type.");
+	}
```

Comment:
```suggestion
	}
	ERR_FAIL_V_MSG(0, "Invalid type.");
```

This will compile ans not suppress the warning if a case is accidentally missed.